### DISCUSSION TITLE
Beta-Release of PK-Sim 10 and MoBi 10

### DISCUSSION THREAD

**Discussion by Yuri05** [Open-Systems-Pharmacology/Forum#945] [Announcement]
# Beta-Release of PK-Sim 10 and MoBi 10
Dear OSP users, 

We are happy to announce that the new Version 10 of PK-Sim and MoBi is now available for beta-testing. The beta-test phase will last until the **end of August**.
The goal of the beta release is to give you an opportunity to test the new features and to let us know if you have any problems or feedback regarding the new implementation.

## How can you test the new features?
You can use the *portable versions* of PK-Sim and MoBi  (download links below). The portable versions do not require installation and can be used in parallel to the official OSP Suite 9.

Your feedback on the new features is highly welcome.

**Important**: The beta-test versions are not final releases of the software packages: please use them for testing purposes only, but not for productive work.

## Downloads:

- [PK-Sim 10 beta portable](http://pk-sim-portable.open-systems-pharmacology.org)

- [MoBi 10 beta portable](http://mobi-portable.open-systems-pharmacology.org)

### New population templates (Chronic Kidney Disease) added

3 CKD population templates added (based on the publication [Malik PRV, Yeung CHT, Advani U, Dije S, Edginton AN. A Physiological Approach to Pharmacokinetics in Chronic Kidney Disease. J Clin Pharmacol 2020. 60 Suppl 1: S52-S62. doi: 10.1002/jcph.1713](https://accp1.onlinelibrary.wiley.com/doi/full/10.1002/jcph.1713)):
* **CKD - Stage 3**: defined as patients with GFR of `31 - 60 [mL/min/1.73m²]`
* **CKD - Stage 4**: defined as patients with GFR of `16 - 30 [mL/min/1.73m²]`
* **CKD - Stage 5**: defined as patients with GFR of   ` 1 - 15 [mL/min/1.73m²]`

Each population includes 1000 individuals aged 18-80 years old, 50% male, based on European population (ICRP)

![](https://user-images.githubusercontent.com/25061876/127352299-413b934e-0a31-4be2-ac1f-adffe80f973a.PNG)

### Modeling of Protein expressions in PK-Sim redesigned and extended
See the [documentation](https://docs.open-systems-pharmacology.org/v/v10/working-with-pk-sim/pk-sim-documentation/pk-sim-expression-data) for details.

![](https://user-images.githubusercontent.com/25061876/127345732-874fac2b-220a-40f5-8f5c-92bcdd59128c.PNG)

* More flexible protein localization settings
  * Enzymes/Binding partners: introduction of **fraction expressed** in different compartments
  * Transporters: introduction of **fraction expressed apical/basolateral** (Kidney/Liver/GI) and **fraction expressed blood brain barrier/tissue** (brain)
* Introduction of **Initial Protein Concentration** as explicit parameter

See [Localizations and initial concentrations of enzymes](https://docs.open-systems-pharmacology.org/v/v10/working-with-pk-sim/pk-sim-documentation/pk-sim-expression-data#localizations-and-initial-concentrations-of-enzymes) and [Localizations, directions, and initial concentrations of transport proteins](https://docs.open-systems-pharmacology.org/v/v10/working-with-pk-sim/pk-sim-documentation/pk-sim-expression-data#localizations-directions-and-initial-concentrations-of-transport-proteins) for the detailed description.

* New transporter types:
  * Plasma <=> Blood Cells
  * Plasma <=> Interstitial
  * Bidirectional

* Transport direction (Influx/Efflux/…) can be set in each organ independently

  ![](https://gblobscdn.gitbook.com/assets%2F-LNl6UIiFP7k0sNQthlR%2Fsync%2F59ea81127a226c02aa651a638378bb80308da5d8.png?alt=media)

### Observed data import in PK-Sim and MoBi redesigned and extended

See the [documentation](https://docs.open-systems-pharmacology.org/v/v10/shared-tools-and-example-workflows/import-edit-observed-data) for details.

* Better support for "Nonmem like" data formats (s. [Supported Formats](https://docs.open-systems-pharmacology.org/v/v10/shared-tools-and-example-workflows/import-edit-observed-data#supported-formats))

* Fully redesigned user interface

  ![](https://gblobscdn.gitbook.com/assets%2F-LNl6UIiFP7k0sNQthlR%2Fsync%2F8e87a7442c2e18566eed7136718bf39da826c8ff.png?alt=media)

* Import configuration can be saved and reused

  ![](https://firebasestorage.googleapis.com/v0/b/gitbook-28427.appspot.com/o/assets%2F-LNl6UIiFP7k0sNQthlR%2Fsync%2F882f7fd4813c22d4dca87ce8d5700c4838b757f7.PNG?generation=1627477017995096&alt=media)

* Previously imported observed data can be **updated** from a new data source

  ![](https://gblobscdn.gitbook.com/assets%2F-LNl6UIiFP7k0sNQthlR%2Fsync%2Fdecc0becc12850ba17ccdf9306d85f9ad81779b7.png?alt=media)

* Extended data filtering

  ![](https://firebasestorage.googleapis.com/v0/b/gitbook-28427.appspot.com/o/assets%2F-LNl6UIiFP7k0sNQthlR%2Fsync%2Fd5c36dd11136d5f29c6e42ae08f7bd86570e33ac.png?generation=1624564037744769&alt=media)

### PK-Sim

* [Bug comparison chart population](https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/1825)
* [Plots: Default background color does not match](https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/1822)
* [Deleting referenced compound template corrupts the template database](https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/1498)
* [Simulations cannot be loaded from snapshot if used Individual or Population Building block was renamed](https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/1662)
* [Export time profile analysis change the units](https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/1640)
* [Renaming protein in individual does not trigger changed state of simulation](https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/1451)
* [Information in advanced protocol is lost in snapshot](https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/1601)
* [Wrong parameter descriptions](https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/1810)
* [The predefined templates for age groups are not supported anymore](https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/1679)
* [PI: "Id not unique"](https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/1299)

### MoBi

* [Tags for event groups](https://github.com/Open-Systems-Pharmacology/MoBi/issues/487)
* [Manual input for event assignment path](https://github.com/Open-Systems-Pharmacology/MoBi/issues/485)
* [Container criteria "in container" only applies for the children of the container](https://github.com/Open-Systems-Pharmacology/MoBi/issues/523)
* [Molecule keyword replacement does not work for global parameter referencing other global molecule parameters](https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/issues/832)
* [Drag and drop of parameter to the "references area" does not work](https://github.com/Open-Systems-Pharmacology/MoBi/issues/507)

### PK-Sim and MoBi

* [PI Corrupted when referencing a process parameter that is removed from the simulation](https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/issues/497)
* [Parameter  Identification: When User close and reopen the PI the results section  shown the new boundaries and not those used in the PI run](https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/issues/818)

**Comment by StephanSchaller** [comment]
Awesome! I will prepare a Tweet ;-)

**Comment by prvmalik** [comment]
Small thing: the CKD template description is missing Shams Ismaeil on the authors list for the reference.

**Reply by Yuri05** [reply]
First bug found 👍

**Comment by arketkar** [comment]
Hi @Yuri05 ,

Yesterday I just tried using v10 beta with a compound setup with active intestinal absorption using transporters. When I ran the simulation, the Fraction dissolved and Fraction Absorbed curves showed negative values. The plasma profile was positive though. When I removed active transport from the model, it started showing positive fractions.
The same project when I tried with Version 8, it showed positive values for fraction dissolved and absorbed.
Sorry right now I do not have the project file with me. However I just thought of sharing this for you to evaluate this ate your end and making any quick bug corrections, if required.

Thanks and regards,
Anant

**Reply by msevestre** [reply]
@arketkar Do you think you can attach that project when you have time?

**Reply by msevestre** [reply]
@Yuri05 Any idea what's going on?

**Comment by arketkar** [comment]
Hi @Yuri05 and @msevestre 

Extremely sorry I am somehow not able to reproduce the scenario in which I got negative fraction dissolved and negative fraction absorbed. I will share that as soon as I could.

However I repeatedly noticed that while making any selection, like the one highlighted in the attached snapshot, it requires 2 to 3 times clicking of mouse to activate the selection. This type of problem I am faced frequently with v10 beta.

Thanks and regards,
Anant
![PK-Sim10Beta_Selection Problem](https://user-images.githubusercontent.com/32693494/131209205-8e52cc9f-4384-44fd-8d1b-3c6187ec5d32.JPG)

**Reply by arketkar** [reply]
Just to update, I am facing this problem of requiring multiple clicks for selecting any parameter even in the Compound building block in v10 Beta. If I have to select one of the multiple Log P entries, then I need to click at least three times to select the right input value.

**Comment by arketkar** [comment]
Hi @Yuri05 and @msevestre 

Please find attached a project file created on 'Version 10 Beta' for your reference. The when I set the k_cat value for Apical Influx to 10, the fraction dissolved and absorbed curves result in negative values. The specific intestinal permeability of compound is kept very low.

Please check if its really a bug or is it something a result of data or model settings, which I am doing wrong. Appreciate your inputs.

Thanks and regards,
Anant
[PK-Sim 10 Beta_Bug Demo.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/7070521/PK-Sim.10.Beta_Bug.Demo.zip)

**Reply by Yuri05** [reply]
@arketkar It's a bug in PK-Sim 10 in the calculation of fraction absorbed into mucosa for some GI active transport types.
Fraction dissolved calculation is the follow-up problem of wrong fraction absorbed.
I will create an issue for this tomorrow. Thank you for the feedback!

**Reply by arketkar** [reply]
Pleasure to be a part of evaluation of Beta version. Good to know that the bug has been identified in right time.

Looking forward to the release of version 10! You all are a great team !!!

Anant Ketkar

**Comment by msevestre** [comment]
@Yuri05 how didn't we catch this in the batch comparison?
